scripts/migrations/11-runtime-cost-adjustment.py

```diff
@@ -0,0 +1,158 @@
+          "send_sir": 141715687500,
+          "send_not_sir": 141715687500,
+          "execution": 102217625000
+        },
+        "add_key_cost": {
+          "full_access_cost": {
+            "send_sir": 101765125000,
+            "send_not_sir": 101765125000,
+            "execution": 101765125000
+          },
+          "function_call_cost": {
+            "send_sir": 102217625000,
+            "send_not_sir": 102217625000,
+            "execution": 102217625000
+          },
+          "function_call_cost_per_byte": {
+            "send_sir": 19715392000000,
+            "send_not_sir": 19715392000000,
+            "execution": 19715392000000
+          }
+        },
+        "delete_key_cost": {
+          "send_sir": 94946625000,
+          "send_not_sir": 94946625000,
+          "execution": 94946625000
+        },
+        "delete_account_cost": {
+          "send_sir": 147489000000,
+          "send_not_sir": 147489000000,
+          "execution": 147489000000
+        }
+      },
+      "storage_usage_config": {
+        "num_bytes_account": 100,
+        "num_extra_bytes_record": 40
+      },
+      "burnt_gas_reward": [
+        3,
+        10
+      ]
+}
+config['runtime_config']['wasm_config']['ext_costs'] = {
+        "base": 265261758,
+        "read_memory_base": 2584050225,
+        "read_memory_byte": 3801396,
+        "write_memory_base": 2780731725,
+        "write_memory_byte": 2723859,
+        "read_register_base": 2493624561,
+        "read_register_byte": 98622,
+        "write_register_base": 2840975211,
+        "write_register_byte": 3801645,
+        "utf8_decoding_base": 3110963061,
+        "utf8_decoding_byte": 289342653,
+        "utf16_decoding_base": 3593689800,
+        "utf16_decoding_byte": 167519322,
+        "sha256_base": 4530046500,
+        "sha256_byte": 24116301,
+        "keccak256_base": 5867223186,
+        "keccak256_byte": 21469644,
+        "keccak512_base": 5798128650,
+        "keccak512_byte": 36651981,
+        "log_base": 0,
+        "log_byte": 0,
+        "storage_write_base": 45187219125,
+        "storage_write_key_byte": 66445653,
+        "storage_write_value_byte": 29682120,
+        "storage_write_evicted_byte": 28939782,
+        "storage_read_base": 32029296375,
+        "storage_read_key_byte": 28463997,
+        "storage_read_value_byte": 3289884,
+        "storage_remove_base": 35876668875,
+        "storage_remove_key_byte": 35342424,
+        "storage_remove_ret_value_byte": 7303842,
+        "storage_has_key_base": 31315025250,
+        "storage_has_key_byte": 28376217,
+        "storage_iter_create_prefix_base": 0,
+        "storage_iter_create_prefix_byte": 0,
+        "storage_iter_create_range_base": 0,
+        "storage_iter_create_from_byte": 0,
+        "storage_iter_create_to_byte": 0,
+        "storage_iter_next_base": 0,
+        "storage_iter_next_key_byte": 0,
+        "storage_iter_next_value_byte": 0,
+        "touching_trie_node": 1,
```

Comment:
Could we make `touching_trie_node` temporarily equal to `storage_read_base`/log_16(20000) ?
Until we do certain optimizations to our trie this operation will have non-zero cost, so for the safety sake we could at least approximate it. The above formula would approximate it.

Reply:
Fixed. Both fixes added to https://github.com/nearprotocol/nearcore/pull/2621.